malaria-ios/NSManagedObjects/MedicineExtension.swift

```diff
@@ -1,34 +1,42 @@
 import Foundation
 
-public extension Medicine{
+extension Medicine {
     /// - returns: `Int`: medicine interval
-    public var interval: Int {
-        get {
-            return Int(internalInterval)
-        }
-    
-        set(value) {
-            internalInterval = Int64(value)
-        }
+    var interval: Int {
+      get {
+        return Int(internalInterval)
```

Comment:
This is example of property to avoid wasting time with conversions everytime the NSNumber/Int64 value is used.